dockerfiles/theia/Dockerfile

```diff
@@ -10,33 +10,44 @@
 FROM node:8-alpine
 
 ARG GITHUB_TOKEN
-ARG THEIA_VERSION=0.3.10
-ENV USE_LOCAL_GIT=true \
-    GITHUB_TOKEN=${GITHUB_TOKEN} \
+ARG THEIA_VERSION=0.4.0-next.b17727c1
```

Comment:
why using a specific version of next instead of next ?
we may want to include all upstream theia fixes on nightlies

Reply:
So you want use next?

Reply:
I do not mind

Reply:
Just to be sure we have working image (I tested it on both docker and oso infras)

Reply:
It will be confused when we move back to the strict Theia version, but dockerhub save "next" tag. @benoitf What do you think?

Reply:
@AndrienkoAleksandr I would think to have a real "nightly" based on "next" and a fixed version as suggested by @mmorhun